webxrlayers-1.bs

```diff
@@ -908,6 +909,11 @@ section in the OpenGL ES 3.0 spec defines these limitations in more detail.
 
 Allocation of the resources for layers (such as memory) MUST be done through the same mechanism as WebGL.
 
+If an {{XRLayer}} is allocated with the {{RGBA}} or {{RGB}} {{XRLayerInit/colorFormat}}, its [=colorTextures=] MUST be exposed as {{RGBA}} or {{RGB}} to the {{WebGLRenderingContext}} context.
```

Comment:
Maybe add a note here that RGBA is the default, and this is equivalent to the behavior of using a XRWebGLLayer without the Layers extension (using its default alpha=true).

Reply:
The spec already calls out that RGBA is the default where it's defining the initialization structures.